### DISCUSSION TITLE
Breakpoint for Navigation to Collapse

### DISCUSSION THREAD

**Discussion by Efrisk3** [bootscore/bootscore#571] [Q&A] [answered]
Bootscore navigation native code does not collapse when viewed on an IPad. How can I change the code to collapse on different screen sizes.

**Comment by crftwrk** [comment, marked as answer]
Navigation collapses on iPad portrait and expands on iPad landscape view. If you want collapsed navbar on iPad landscape as well, you have to change `navbar-expand-lg` to `navbar-expand-xl` and toggler `d-lg-none` to `d-xl-none` in your `header.php`.

Check here https://bootscore.me/documentation/header-php/#Breakpoints

**Reply by Efrisk3** [reply]
Solved. Thank you!
